text/0000-edition-based-method-disambiguation.md

```diff
@@ -0,0 +1,224 @@
+- Feature Name: `edition_based_method_disambiguation`
+- Start Date: 2022-03-04
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a way to introduce new trait methods that conflict with
+pre-edition[^1] trait methods in downstream crates in a backwards compatible fashion.
+We do so by annotating new methods with the edition they're introduced in. Then
```

Comment:
Something which isn't clear to me from the RFC is what this annotation is meant to look like. Even if we may not expect users to use this, as someone who occasionally contributes to the stdlib I'd like to understand what this looks like from the implementer's perspective. Could we add an example of this to the "guide level explanation" section?

Reply:
Oh actually now realizing that the "reference" section describes this in more detail. That actually helps answer some of the questions I had.